specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2021-09-01-preview/ThreatIntelligence.json

```diff
@@ -56,9 +56,6 @@
           {
             "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
           },
-          {
-            "$ref": "../../../common/2.0/types.json#/parameters/OperationalInsightsResourceProvider"
```

Comment:
>0/types.json#/parameters/OperationalInsightsResourceProvider" [](http://example.com/codeflow?start=39&length=61)

Do you need to remove this type as well ?

Reply:
Yes, thank you!